spec/checkPullRequestTemplateSpec.js

```diff
@@ -494,12 +653,58 @@ describe('Pull Request Template', () => {
         repo: payloadData.payload.repository.name,
         owner: payloadData.payload.repository.owner.login,
         body:
+        'Hi @' +
+        payloadData.payload.pull_request.user.login +
+        ', can you complete the following:' + '\n' +
+        '1. ' + incompleteChecklistSection.karma_linter +
+        '\n' + '2. ' + incompleteProofOfChangesSection + '\nThanks!',
+      });
+    });
+  });
+
+  describe('testing generateComment function with bodyWithInvalidTemplate',
```

Comment:
@jameesjohn PTAL if these tests are correct. I'll do the rest of the tests after confirmation since the rest of the tests will follow the same format.

Reply:
Yes, it looks good.